src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -596,6 +596,14 @@ static constexpr Property<ExecutionMode> execution_mode{"EXECUTION_MODE_HINT"};
 static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization_group_size{
     "DYNAMIC_QUANTIZATION_GROUP_SIZE"};
 
+/**
+ * @brief This property defines unsupported group size for dynamic quantization optimization
+ * @ingroup ov_runtime_cpp_prop_api
+ *
+ * If dynamic_quantization_group_size is larger than this unsupported value, dynamic quantization will be disabled.
+ */
+static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization_group_size_unsupported{
```

Comment:
Could you please clarify who should be responsible for setting this property? If it should be set during model conversion/compression, why not just set the appropriate dynamic_quantization_group_size value? Or, if it's the user's responsibility, why can't they disable dynamic_quantization explicitly?

Reply:
I will not introduce any heuristic to check this automatically. This will be set based on performance/accuracy validation result. For iGPU, we are seeing perf gain with gs=32 and it shows on-par performance with f16 activation. So we don't need to take care of such case.

Reply:
@sshlyapn updated PR to move the option into GPU plugin. Also, changed the option name to have `GPU_` prefix.